### DISCUSSION TITLE
Force osrm to route from a different way for a specific location

### DISCUSSION THREAD

**Discussion by Cactustory** [Project-OSRM/osrm-backend#6800] [Q&A]
There are two ways for a specific location, by default it uses the better way, How Can I force osrm to use the second way just for this specific location?
I know it's normal to use the best way, but I want to learn how to do it for my project.

**Comment by frodrigo** [comment]
OSRM can provide an alternative way. When it is possible.

You can also change the definition of the best way in the profile to get an other route according to other criteria.
